checker/src/main/java/org/checkerframework/checker/optional/OptionalVisitor.java

```diff
@@ -491,6 +635,23 @@ public Void visitVariable(VariableTree tree, Void p) {
     return super.visitVariable(tree, p);
   }
 
+  /**
+   * Given a variable declaration, add the enclosing method in which it is found (if one exists) to
```

Comment:
This says "given a variable", but the method name is more specific.  Please make them consistent.